### ISSUE TITLE
Get rid of the term "combined round"

### ISSUE THREAD

**Issue by timreyn** [thewca/wca-regulations#543]
I've heard many competitors and organizers confused by the terms "combined round" and "combined final".  One common misunderstanding (among both newer competitors and people who have been around a while) is that "combined final" means an event with only one round, so everything is combined into the final.

I don't really know what the benefit of the term is.  A combined final is just a final that has a cutoff.  But I don't know why our software and regulations need to treat the two separately.  It's redundant to say that a round is a combined round AND list a cutoff -- saying that the round is combined doesn't add any information, and clutters the schedule and list of events.

I propose, as an alternative, to just say that rounds may have cutoffs, and that people must meet these cutoffs after 2 solves (for an average of 5) or 1 solve (for a mean of 3) in order to finish their average.

This would not be a functional change, just a terminology change.

**Comment by jfly** [comment]
I am very supportive of this change!

**Comment by lgarron** [comment]
I agree that "combined" is not the best word, but we should still have a name to refer to these. Do you have something more direct in mind, e.g. "2-phase round"?

I would favor "cutoff round", but I'd be worried that it "cutoff" could be confused with "time limit" again. (After all, the time limit is when a solve is "cut off", but the cutoff is the "time limit"' you need to meet to advance to the next phase.)

(Also note that the Regulations don't mentioned a "combined final".)

**Comment by timreyn** [comment]
I don't agree that we should have a name for these rounds. We should just call them "rounds". IMO it would be much cleaner to say that a round has a cutoff than to call it a "cutoff round".

When combined rounds were introduced, it made sense to distinguish them from "normal" rounds, since the concept was new. Now cutoffs are very common, and so there's less need to highlight the fact that there's a cutoff.

**Comment by timreyn** [comment]
@lgarron, could you explain why you feel there should be a term for such rounds?

**Comment by lgarron** [comment]
> @lgarron, could you explain why you feel there should be a term for such rounds?

We need a way to refer to them in the Regulations.

If you think "round with a cutoff" is sufficient, we could use that. But I suspect that will informally get shortened to "cutoff round" unless we proactively choose a more accurate name.

**Comment by jfly** [comment]
I don't see how this is any different than time limits. Does "round with time limit" get shortened to "time limit round" anywhere?

**Comment by lgarron** [comment]
> I don't see how this is any different than time limits. Does "round with time limit" get shortened to "time limit round" anywhere?

[All rounds have time limits](https://www.worldcubeassociation.org/regulations/#9s), but not all rounds have cutoffs.

**Comment by timreyn** [comment]
There are only 3 regulations and 2 guidelines that refer to combined rounds.  I don't think that's enough that a lack of a name would be a problem.

9o could be completely dropped, for example, since there wouldn't be any need to clarify that a round counts as one round.

**Comment by julesDesjardin** [comment]
I agree with this change, and leaving those rounds without a name doesn't seem much of a problem to me. We can change the phrasing of the regulations to use "if the round has a cutoff, ..." or something similar.
In competitions we refer to all rounds as simply "rounds", and then the organizer specifies if there is a cutoff or not. I've never seen anyone speak about "combined rounds" anywhere else than in the regulations (maybe it's cultural specific ?). So I don't think they will be called "cutoff rounds" (at least in European countries I've been to).

**Comment by Laura-O** [comment]
This issue is confusing me. I frequently use the term "combined round", especially to speak about whether the round has a cutoff or not.

> When combined rounds were introduced, it made sense to distinguish them from "normal" rounds, since the concept was new. Now cutoffs are very common, and so there's less need to highlight the fact that there's a cutoff.

I don't think the fact that something is common justifies to remove a term that describes it. While I agree that it's not the best word, I think we definitely need a word for it.

**Comment by timreyn** [comment]
@Laura-O is there an advantage to saying "this is a combined round" instead of "this round has a cutoff"?  I think the latter is much clearer, because newer competitors (or more experienced competitors who are confused) don't need to learn an additional piece of terminology.

**Comment by lgarron** [comment]
@thewca/wrc-team I don't feel *strongly* that we need a word for these rounds, but Laura does. Anyone else?

**Comment by Laura-O** [comment]
> @Laura-O is there an advantage to saying "this is a combined round" instead of "this round has a cutoff"? I think the latter is much clearer, because newer competitors (or more experienced competitors who are confused) don't need to learn an additional piece of terminology.

"Combined round" is a term for a round with a cutoff. "This round has a cutoff" is the explanation. I think "Combined round" is much easier to grasp when it's mentioned on a schedule (if you know the definition, yes).

Additionally, if we remove this from the regulations, how would we handle this on the results pages?
I think changing "3x3x3 One-Handed Combined Final" to "3x3x3 One-Handed Final with a cutoff" is not an improvement. If we still call it "Combined Final" and remove it from the regulations, there is no definition for this term. If we call it just "Final", we remove either the information why some competitors have fewer results than others or the information that all competitors made the cutoff.

**Comment by pedrosino** [comment]
Maybe we should list thr cutoffs in the results page?

Someone who has no clue about the WCA will see "combined" something and some results missing and won't understand. I think not many of those will read the regulations carefully to find the definition.

**Comment by AlbertoPdRF** [comment]
I think the term "Combined" is very useful and makes all the sense in the world, so I wouldn't remove it honestly.

**Comment by jfly** [comment]
I agree with Pedro. I'd like us to just say "final round", and then actually list the cutoff there. This is something we have don't before because we didn't have that data, but now we're getting close to requiring that delegates provide this data when submitting results. We could even provide an explanation of a cutoff right there, or at least link to an explanation.

@albertopdrf, I don't find the term intuitive, and I think it's clear from this discussion that at least some other people also find it unintuitive.

I also don't think the term is particularly useful, because I think it's rare to want to know if a round has a cutoff, but you don't care what the cutoff is. If you're going to go on to say what the cutoff is *anyways*, I don't see the value in calling it a combined round versus just a round.

**Comment by KitClement** [comment]
Most organizers I've talked with do not understand the idea of what a "combined" round is. As Tim mentioned, most think a combined final is "combined" since it's an event with only one round -- many don't realize that "combined first round" is also a valid format name. But this is hardly surprising to me, given that the term "combined" is an artifact of when competition structures used to have a best of X round, then an average of 5 round. 10+ years ago, it made sense to call a round combined, since it was literally a commonly used combination of these two round formats into one round. But since we don't do this now, are we really "combining" anything? Cutoff times have just become a standard practice, and I don't think anyone describes making the cutoff time as advancing to the next round/phase of a combined round.

I'd be in support of dropping the "combined" term. 9g/9g2 can be reworded without defining a term for the round, as Jules mentioned.

**Comment by SAuroux** [comment]
A "combined round" is not only a round with a cutoff, it is a round that combines two different formats in one round. For me, it makes all sense in the world to keep a well-defined term for this, especially from the database/results perspective that @Laura-O already mentioned above.

Besides all that...this really is not meant as a personal/regional attack, but there is simply no way around noticing that everyone arguing against the term here comes from the region that already had huge problems in using the term "cutoff" appropriately. If the correct WCA terminology has not even been used in an area up to recently, it is no surprise that competitors/organizers are rather confused by the term "combined round".

**Comment by julesDesjardin** [comment]
I agree with @pedrosino, something like "Final [smaller] (cutoff : 30 seconds)" would be perfect (and also adds some information). (We could also add the time limit while we're at it)
For older rounds it can be "cutoff : unknown" (or we can find a way to detect it (won't work in all cases), e.g. if people with 29 got an average but not people with 30), and for non combined either "cutoff : none", or don't show this field.
I was worried about how it may not be possible to implement, but Jeremy's post shows that it's not a problem !

I also agree with @KitClement, we're really not "combining" anything, the way we present it (informally in competitions, not in the regulations) is usually "you need to do better than X to be allowed to finish your average", not "you need to do better than X to advance to the next phase, which is an average" so it's more of a restriction than a combination.

**Comment by timreyn** [comment]
@SAuroux The US is large.  We've been using terms correctly at my competitions for a long time, and competitors and spectators at those competitions are still very confused by the term "combined round".

**Comment by pedrosino** [comment]
@SAuroux also we never used wrong terms here in Brazil...
@julesDesjardin is from France and I don't recall problemw with wrong terminology there as well.

**Comment by kingmathyall** [comment]
I personally think that simply having a schedule that displays the cutoffs provides the competitor with enough information about a specific round format. If there is a cutoff list, then great, the competitor knows there is a cutoff for the first 2 solves (or 1 for mo3). If there is no cutoff listed, then there is no cutoff for that round.

**Comment by lgarron** [comment]
@thewca/wrc-team Does anyone feel strongly about this? I would prefer to change "combined round" to "cutoff round" for simplicity and consistency.

I don't plan to revisit #544 soon unless more of us are in favor.

**Comment by hanwu85** [comment]
Just some information.
In our translated version (Traditional Chinese) of the this term, we use "門檻制" round.
A 門檻 is the thing on the bottom of a door frame that prevent stuffs(water, dust...) from coming in.
  (See pictures [here](https://www.google.com.tw/search?q=%E9%96%80%E6%AA%BB&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjU_L3D--3kAhUZzIsBHaKbDwgQ_AUIEigB&cshid=1569483098788570&biw=1858&bih=977)) You have to be higher than the 門檻 to going through(like lifting up your feet).  A 門檻 is literally translated to "threshold"

If we use threshold-round, does it make sense for native speakers?

**Comment by lgarron** [comment]
> If we use threshold-round, does it make sense for native speakers?

I think that "cutoff round" is pretty close!

**Comment by Sixstringcal** [comment]
I don't think the term "combined round" should be removed.  That being said, nobody I ever work with for comps seems to understand what it is.  Organizers I've worked with have very frequently just called all finals "combined finals" and assumed that's how it worked.  I know when I started organizing, I was under the impression that that was how it worked as well.  I think that it might be best to just change the term.  Not sure to what, but maybe something like a "cutoff round" (granted, it doesn't sound as good as "combined final").

I also don't see it as a huge loss if the term is thrown out.
